README.md

```diff
@@ -0,0 +1,88 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Rails Back End
+
+> Understand the setup process for adding React/Redux to rails application.
+
+This project is made as part of Microverse program.
+
+you can view the front-end of this app [here](https://github.com/oluyaratosin123/Hello-React-Front-End)
+
+## Project Requirement
+
+ - Create a new Rails API app called 'hello-rails-back-end'.
+ - Initialize your project with Git.
+ - Make sure that your project has a Postgres database set up. 
+ - Create a table for storing your messages and create 5 different greetings. 
+ - Create an API endpoint that selects a random greeting from your table (you will need a controller with an action an rails route).
+
+## Built With
+
+- RUBY
+- RAILS
+- GEM
+- TERMINAL
+- React/Redux
+
+## Getting Started
+
+Start by cloning the repository.
+
+```
+git clone git@github.com:oluyaratosin123/Hello-Rails-Back-End.git
+```
+
+Navigate to the created folder.
+
+```
+cd Hello-Rails-Back-End
+```
+
```

Comment:
- [ ] These are not the correct link and directory for cloning the repo. Kindly correct that.